docs/modules/ROOT/pages/tutorial-kubernetes.adoc

```diff
@@ -1,73 +1,171 @@
-= Kubernetes example
+= CMOS Kubernetes Deployment
 
 [abstract]
-A simple working example from scratch locally using https://kind.sigs.k8s.io/[Kubernetes-IN-Docker (KIND)^] with the https://docs.couchbase.com/operator/current/overview.html[Couchbase Autonomous Operator (CAO)^] deploying Couchbase Server then being monitored by CMOS.
+A simple working example from scratch in kubernetes cluster with the https://docs.couchbase.com/operator/current/overview.html[Couchbase Autonomous Operator (CAO)^] deploying Couchbase Server then being monitored by CMOS.
 
 include::partial$tutorial.adoc[]
 
+== Overview
+
+CMOS is a simple, out-of-the-box solution based on industry standard tooling to observe the state of your Couchbase cluster.
+
+== Deploy CMOS
+
+=== kube-state-metrics
+
+A prerequisite to configuring CMOS is to install kube-state-metrics. This can be done via helm.
+
+[source,console]
+----
+$ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+$ helm repo update
+$ helm upgrade --install kube-state-metrics prometheus-community/kube-state-metrics
+----
+
+=== Prometheus
+
+We provide Prometheus configuration via a Kubernetes `ConfigMap` which contains all the details for credentials, scrape targets, etc.
```

Comment:
Let us change the config map name to `prometheus-config-cmos`, in this way we can avoid saying that delete the configmap.